guiEditor/src/guiNodeTools.ts

```diff
@@ -91,6 +91,13 @@ export class GUINodeTools {
                 element.color = "transparent";
                 element.isPointerBlocker = true;
                 return element;
+            case "ButtonImage":
+                element = new Image("Image", "https://playground.babylonjs.com/textures/grass.png");
```

Comment:
could the url be moved as a static so that users could change it if they want to run everything locally for instance ?

Reply:
Wouldn't the users be able to use the image in the ui afterwards?